pkgs/development/python-modules/aioredis/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, fetchPypi
+, buildPythonPackage
+, hiredis, async-timeout
+}:
+buildPythonPackage rec {
+
+  pname = "aioredis";
+  version = "1.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "84d62be729beb87118cf126c20b0e3f52d7a42bb7373dc5bcdd874f26f1f251a";
+  };
+
+  propagatedBuildInputs = [
+    hiredis
+    async-timeout
+  ];
+
```

Comment:
I would explicitly set `doCheck = false` with an explanatory comment that it relies on the presence of a redis server.

Reply:
I got that added. Thanks for the tip.